src/Service/UserBlocker.php

```diff
@@ -34,7 +34,7 @@ public function __construct( MediawikiApi $api ) {
 	 * @throws InvalidArgumentException
 	 * @return bool
 	 */
-	public function block( $user, BlockOptions $options = null ) {
+	public function block( $user, BlockOptions $options ) {
```

Comment:
You would need to replace "BlockOptions $options" with "$options" in that case, as otherwise the class would 'fail' because the $options argument is not a BlockOptions object.